engine/src/player-platform.cpp

```diff
@@ -1384,7 +1384,9 @@ static bool MCPathIsRemoteURL(const char *p_path)
 {
```

Comment:
A really minor thing - but MCPathIsRemoteURL is no longer returning true for a 'remote url' - it is just returning true for a URL. If the function isn't used elsewhere (I don't believe it is) then it should probably be renamed 'MCPathIsURL'.